README.md

```diff
@@ -18,6 +18,25 @@ Install the dependencies and start a local instance using the following script:
 yarn start
 ```
 
+## Deploying the UI
+
+1. Change `index.html` to point to `dist/index.js` (instead of `example.js`) and upload to s3 bucket (or other file hosting service).
+2. Upload `example.css`.
+3. Build the js/css bundle (which will appear in the `dist/` directory) and upload.
+
+Here's an example set of commands using the AWS CLI:
+```bash
+# The first two steps only need to be performed the first time.
+aws s3 cp index.html s3://my-bucket --acl public-read
+aws s3 cp example.css s3://my-bucket --acl public-read
+# Anytime the otp-react-redux code changes, you'll need to run the following steps:
+yarn build
+aws s3 cp --recursive dist/ s3://my-bucket/dist --acl public-read
```

Comment:
It might be better IMO to tell the reader to upload the files organized as follows:
- index.html (modified to point to `dist/index.js`
- example.css
- dist
    - index.js
    - index.js.map
    - index.css
    - index.css.map

Or even this, and just upload the contents of dist folder:
- dist
    - index.html (modified to point to `index.js`
    - example.css
    - index.js
    - index.js.map
    - index.css
    - index.css.map